mock/api.js

```diff
@@ -29,11 +29,16 @@ export const mockResults = {
           controller_dir: casual.word,
         },
         run: {
+          key: index,
```

Comment:
What if we used the id field that contains a generated uuid as the key for rendering?

Reply:
@gurbirkalsi we will need the `key` field for rendering. In the case of multiple selection, we are keeping track of `selectedRowKeys` which is an internal function requiring keys for each row. However I have fixed the same by assigning same value to `key` as `id`